pkg/plugins/platform/local.go

```diff
@@ -674,6 +675,10 @@ func (l *LocalApp) Stop() error {
 		return fmt.Errorf("no site to remove")
 	}
 
+	if strings.Contains(l.SiteStatus(), SiteDirMissing) || strings.Contains(l.SiteStatus(), SiteConfigMissing) {
+		return fmt.Errorf("ddev can no longer find your application files at %s. If you would like to continue using ddev to manage this site please restore your files to that directory. If you would like to remove this site from ddev, you may run ddev rm %s", l.AppRoot(), l.GetName())
```

Comment:
Please wrap "ddev rm <sitename>" in quotes:

`Failed to stop containers for wordpress: ddev can no longer find your application files at /Users/beeradb/Projects/ddev/wordpress. If you would like to continue using ddev to manage this site please restore your files to that directory. If you would like to remove this site from ddev, you may run ddev rm wordpress`